contrib/playlib/test/src/mill/playlib/PlayTestsCommon.scala

```diff
@@ -0,0 +1,25 @@
+package mill.playlib
+
+import mill.util.{TestEvaluator, TestUtil}
+import utest.framework.TestPath
+
+trait PlayTestSuite {
```

Comment:
I'd like to have filename and content in sync. Please rename the file to `PlayTestSuite.scala`.

Reply:
Ops, this was originally `object PlayTestsCommon` 😄
I'm renaming it!